packages/experience-builder-sdk/src/core/designTokenRegistry.ts

```diff
@@ -2,13 +2,44 @@ import { OUTGOING_EVENTS } from '../constants';
 import { sendMessage } from '../communication/sendMessage';
 import { DesignTokensDefinition } from '../types';
 
+const designTokensRegistry = {} as DesignTokensDefinition;
+
 /**
  * Register design tokens styling
  * @param designTokenDefinition - {[key:string]: Record<string, string>}
  * @returns void
  */
 export const defineDesignTokens = (designTokenDefinition: DesignTokensDefinition) => {
+  Object.assign(designTokensRegistry, designTokenDefinition);
   sendMessage(OUTGOING_EVENTS.DesignTokens, {
     designTokens: designTokenDefinition,
   });
 };
+
+export const getDesignTokenRegistrationForSpacing = (breakpointValue: string) => {
+  let resolvedValue = '';
+  const values = breakpointValue.split(' ');
+  values.forEach((value) => {
+    let tokenValue = value;
+    if (isTemplateStringFormat(value)) tokenValue = resolveSpacingDesignToken(value);
+    resolvedValue += `${tokenValue} `;
+  });
+
+  return resolvedValue;
+};
+
+// Using this because export const StringTemplateRegex = /\${(.*?)\}/g doesn't work
+const isTemplateStringFormat = (str: string) => {
+  if (!str || str.length < 3) return false;
+  return str.charAt(0) === '$' && str.charAt(1) === '{' && str.charAt(str.length - 1) === '}';
+};
```

Comment:
The regex you added to the `resolveSpacingDesignToken` method could be used here if you wanted
```suggestion
const isTemplateStringFormat = (str: string) => {
  const regex = /\$\{([^}]+)\}/g;
  return regex.test(str);
};
```